tailwind.config.mjs

```diff
@@ -89,6 +89,7 @@ export default {
         base: ['13px', '1.4'], // Current base font size for sidebar
         lg: ['14px'],
         xl: ['16px'],
+        '2xl': ['18px'],
```

Comment:
One issue that is a bit annoying when reviewing Tailwind code is that it isn't trivial to look up what a particular class does from the GitHub editor (I was looking up `text-2xl` earlier). I have just installed the Tailwind extension in VS Code but am not seeing intellisense for the `className` attribute. Can you remember if any additional configuration was required to make it work?

Reply:
I'd been getting along fairly well with it (Intellisense extension in VSCode). This comment prompted me to look into it again, but alas:

<img width="841" alt="image" src="https://user-images.githubusercontent.com/439947/166232703-cfced892-0429-45cb-a3e1-b672836682a5.png">

Any bright ideas? (We use an `.mjs` extension on our tailwind configs).